spec/services/images/optimizer_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe Images::Optimizer, type: :service do
+  include CloudinaryHelper
```

Comment:
Why do we have to include this in the spec but not in the actual image optimizer file?

Reply:
This is so I call `cl_image_path` as supposed to  `ActionController::Base.helpers.cl_image_path`. 

I'm trying to slim down available methods by removing `include CloudinaryHelper` from the app but I figure it doesn't too much in spec (which also let me access other view helpers like `asset_path`). This spec file is my current playground to verify we're still generating the proper URL. I can remove this in upcoming PRs to reduce indirection.